pkg/roachpb/api.proto

```diff
@@ -1606,12 +1606,20 @@ message ExportResponse {
 // ranges that cannot be moved will include an error detail in the response and
 // won't fail the request.
 message AdminScatterRequest {
+
+  enum Class {
+    ARBITRARY = 0;
+    ORGANIZATION = 1;
```

Comment:
nit: this should probably be ingestion, not organization, since pretty much the only reason to call scatter is to balance the placement of ranges prior to ingestion or starting a workload, not to achieve any specific organization of the replicas.

Reply:
Updated.